Docker image upload is failing

The GHA [docker upload workflow](https://github.com/IntersectMBO/cardano-db-sync/actions/runs/7757112538/job/21246061697) is failing on the latest tag:

```
 The push refers to repository [ghcr.io/intersectmbo/cardano-db-sync]
190fe373008b: Preparing
6a7775dbe07b: Preparing
ca8fa9811680: Preparing
denied: permission_denied: write_package
Error: Process completed with exit code 1.
```

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -2,7 +2,7 @@ version: "3.9"
 
 services:
   postgres:
-    image: postgres:11.18-alpine
+    image: postgres:14.10-alpine
     environment:
       - POSTGRES_LOGGING=true
       - POSTGRES_DB_FILE=/run/secrets/postgres_db
@@ -32,7 +32,7 @@ services:
         max-file: "10"
 
   cardano-node:
-    image: inputoutput/cardano-node:8.0.0
+    image: ghcr.io/intersectmbo/cardano-node:8.7.3
     environment:
       - NETWORK=${NETWORK:-mainnet}
     volumes:
@@ -53,7 +53,7 @@ services:
         max-file: "10"
 
   cardano-db-sync:
-    image: inputoutput/cardano-db-sync:13.2.0.0
+    image: ghcr.io/intersectmbo/cardano-db-sync:13.2.0.0
     environment:
       - DISABLE_LEDGER=${DISABLE_LEDGER}
       - NETWORK=${NETWORK:-mainnet}

--- a/docker-test.yml
+++ b/docker-test.yml
@@ -2,7 +2,7 @@ version: "3.5"
 
 services:
   postgres:
-    image: postgres:11.5-alpine
+    image: postgres:14.10-alpine
     environment:
       - POSTGRES_LOGGING=true
       - POSTGRES_DB_FILE=/run/secrets/postgres_db
@@ -33,4 +33,3 @@ secrets:
 
 volumes:
   postgres:
-

